ISLE/isleapp.h

```diff
@@ -96,6 +96,7 @@ class IsleApp {
 };
 
 extern IsleApp* g_isle;
+extern MxS32 g_closed;
```

Comment:
I tried playing with this to look at apt hook exiting, should be removed before merging in case we find a solution to the exit crash